### ISSUE TITLE
Split archive

### ISSUE THREAD

**Issue by sepehr1014** [adamhathcock/sharpcompress#60]
How can we compress one file and split it to multiple parts?

**Comment by sepehr1014** [comment]
If this can't be done, can you point out which parts need to be modified in order to implement this feature?

**Comment by tanapop** [comment]
I need this solution too.

**Comment by adamhathcock** [comment]
The code to compress RAR isn't open source.  I haven't implemented Zip splitting.  You're welcome to contribute back that.

**Comment by jochenvangasse** [comment]
Any progress on this?

**Comment by adamhathcock** [comment]
No.  Apparently, DotNetZip does it so it could be ported from there.
